github/ci/prow-deploy/kustom/base/configs/current/plugins/plugins.yaml

```diff
@@ -327,6 +327,25 @@ plugins:
   - trigger
   - override
 
+  RedHatQE/openshift-python-wrapper:
```

Comment:
Could you please move the array elements two spaces to the left, so that they are aligned with the repo name? Just a matter of uniformity, same as we have with the rest repos.

Reply:
ACK